libs/game/info.ts

```diff
@@ -252,7 +252,12 @@ namespace info {
             allScores = [];
         }
 
-        settings.writeJSON(allScoresKey, allScores);
+        const scoresObj = {
+            "allScores": allScores,
+            "scoringType": allScores.length ? scoringType : "None"
+        }
+
+        settings.writeJSON(allScoresKey, scoresObj);
```

Comment:
I'm not totally familiar with how setting/reading these settings works, but will we hit compatibility issues with games that are shared before the update which someone then tries to play or import into kiosk after the update?

If it's just an issue with importing into kiosk, that may be fine. But if it somehow breaks the game loading at all, that'd probably be an issue.

Reply:
If it does cause issues, a potential fix could be to save this with a different key and change the loading logic to account for both (giving this one priority)

Reply:
I might be misunderstanding your question, but I made sure to have backwards compatibility on the kiosk side. If the JSON that is received by the kiosk doesn't have an entry for "allScores" we just parse the JSON and use the entry like we did before.

Reply:
Oh, and it's important to note that this key is not being used anywhere else, only for kiosk.

Reply:
When we release arcade, both kiosk page and simulator are released together. I don't see a compat issue even if the formats of communication change. I don't know of any other host using these at this moment other than kiosk

Reply:
@abchatra I was more concerned with games shared before the update being imported after the update (like games on the home page, for example).

But from Sarah's comments, seems like it'll be fine regardless. Thanks for the info!